### PR TITLE
First Web API: Linux: Add tools path

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes: #30439

Added Linux instructions for adding .NET tools directory to the system path.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/tutorials/first-web-api.md](https://github.com/dotnet/AspNetCore.Docs/blob/07ffb01b01afece02943404e1c203d2b92dcba6a/aspnetcore/tutorials/first-web-api.md) | [Tutorial: Create a web API with ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?branch=pr-en-us-32683) |

### CODE REVIEW

File: aspnetcore/tutorials/first-web-api.md [hunk 1]
```diff
@@ -404,7 +411,7 @@ The previous section showed an example of the `/api/todoitems/{id}` route.
 
 Follow the [POST](#post7) instructions to add another todo item, and then test the `/api/todoitems` route using Swagger.
 
-This app uses an in-memory database. If the app is stopped and started, the preceding GET request will not return any data. If no data is returned, [POST](#post7) data to the app.
+This app uses an in-memory database. If the app is stopped and started, the preceding GET request won't return any data. If no data is returned, [POST](#post7) data to the app.
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-web-api.md:414]:
```suggestion
This app uses an in-memory database. If the app is stopped and started, the preceding GET request doesn't return any data. If no data is returned, [POST](#post7) data to the app.
```

File: aspnetcore/tutorials/first-web-api.md [hunk 2]
```diff
@@ -321,6 +321,13 @@ The preceding commands:
 * Add NuGet packages required for scaffolding.
 * Install the scaffolding engine (`dotnet-aspnet-codegenerator`) after uninstalling any possible previous version.
 
+For Linux, depending on your configuration, the .NET tools directory may need to be added to the system PATH. For common configurations, the .NET tools directory is automatically added to the system PATH during installation. The .NET tools directory can be added to the system path with the following command:
```
Comment (tdykstra) [inline aspnetcore/tutorials/first-web-api.md:324]: "system path" appears once, "system PATH" twice.

**Review by tdykstra — APPROVED** [review]
Looks good.

File: aspnetcore/tutorials/first-web-api.md (hunk 1 above)
Comment (wadepickett) [reply aspnetcore/tutorials/first-web-api.md:414]: Thanks.  Acrolinx wanted to use "won't" but that did seem off.  You are right.

**Pull request merged**